### DISCUSSION TITLE
Version 8.4 beta 3

### DISCUSSION THREAD

**Discussion by Arnova8G2** [Arnova8G2/Gcam_8.7.250.44#68] [Ideas]
**Change logs:**

- Fixed bugs with noise mode
- Fixed bugs after clearing cache
- Fixed use camera logical in main camera
- Saber upgrade for better use
- Fixed GID now works correctly
- Bug fix on some device

**Help for Sabre**

- Best choice for sabre and used sabre + spatial_rgb
camera.sabre.enabled **ON**
camera.sabre.spatial_rgb_enabled **ON**
camera.sabre.allowed **ON**

- Default by google
camera.sabre.enabled                works from 1.2x zoom
camera.sabre.spatial_rgb_enabled    works from 2.5x zoom
camera.sabre.allowed                always ON even without zoom

- If a camera crashes after capture use unzoomed which corresponds to cam or mode used
camera.sabre.unzoomed_wide          disable main cam
camera.sabre.unzoomed_ultratele     disable ultratele cam
camera.sabre.unzoomed_ultrawide     disable ultrawide cam
camera.sabre.unzoomed_front         disable front cam
camera.sabre.unzoomed_gouda         disable portrait
camera.sabre.unzoomed_astro         disable astro
camera.sabre.unzoomed_ns            disable night

- To be used if no compatibility with sensor
camera.sabre.disable_ultratele      force disable all sabre for ultratele cam
camera.sabre.disable_ultrawide      force disable all sabre for ultrawide cam

- A single selection only otherwise all sabre will be deactivated, not to be used
camera.sabre.force                  force use sabre
camera.sabre.spatial_rgb_force      force use spatial_rgb
camera.sabre.wiener_force           force wiener

**Help for GID (Google ID cam)** 

- If capture has a lot of color noise change gid
0   is wide cam
10  is wide cam Binned
11  is wide cam Remosaicked

4   is ultratele cam 
7   is ultratele cam Binned
12  is ultratele cam Remosaicked

8   is ultrawide cam
9   is ultrawide cam Binned

1   is front cam 
2   is front cam Ultrawide
6   is front cam Infrared
13  is front cam Secondary

APK certificate fingerprints: 
SHA-1: e0be67031a86c4178239256f80d631d94451e263 

APK file name: **arn.android.gcam.beta**  
Gcam-8.7.250.build-8.4Beta3.230623.0813.apk 
 
APK file hashes: 
MD5: a6917037730f954defb8ff75ce814441 
SHA-1: f6f18ea46aa4813b061cd107513329afbf041213 

<hr /><em>This discussion was created from the release <a href='https://github.com/Arnova8G2/Gcam_8.7.250.44/releases/tag/V8.4-Beta3'>Version 8.4 beta 3</a>.</em>

**Comment by myselflatu** [comment]
Having issues with AWB loading for example if i load awb for multiple lenses only one awb shows up check the screenshot below.
![Screenshot_20230624-121932_Camera](https://github.com/Arnova8G2/Gcam_8.7.250.44/assets/40817763/54d0da16-a465-47c2-b85d-0fb940dbaade)
